octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -825,8 +825,11 @@ def list_reports(
             query = query.where(ReportRecipe, recipe_id=str(recipe_id))
             query = query.where(Report, report_recipe=ReportRecipe)
 
-        count_results = self.query(query.count(), valid_time)
-        count = 0 if not count_results else count_results[0]
+        count = 0
+
+        if not ignore_count:
+            count_results = self.query(query.count(), valid_time)
+            count = 0 if not count_results else count_results[0]  # type: ignore
```

Comment:
Why should this type error be ignored? What kind of type error are you getting?

Reply:
An assignment error due to the response having different options in its structure, but putting this in an if-else worked (though at this point mypy is dictating how to write flow of control unless we add bulke type annotations on basic assignments 🤔)